websocket_proxy.go

```diff
@@ -33,7 +33,7 @@ func websocketHTTPConnect(proxy, urlString string) (net.Conn, error) {
 
 	cc := httputil.NewProxyClientConn(p, nil)
 	cc.Do(&req)
```

Comment:
I think you need to change this to `response, err := cc.Do(&req)` or else the error check is just a repetition of line 23 right?